src/many-identity/src/identity/cose.rs

```diff
@@ -44,7 +44,7 @@ impl Signature for CoseKeyIdentitySignature {
 
 #[derive(Clone, Debug, PartialEq)]
 pub struct CoseKeyIdentity {
```

Comment:
This will implement that trait. I'm planning on splitting `CoseKeyIdentity` into multiple types; `Ed25519Identity`, `EcDsaIdentity` and `HsmIdentity`. So in a sense, we will get rid of this schizophrenic type.